lib/src/bound_multipart_stream.dart

```diff
@@ -8,7 +8,7 @@ import 'dart:convert';
 import 'char_code.dart' as char_code;
 import 'mime_shared.dart';
 
-/// Bytes for '()<>@,;:\\"/[]?={} \t'.
+/// Bytes for `()<>@,;:\\"/[]?={} \t`.
```

Comment:
This linted on CI (via the dartdoc HTML lint), but not locally. 🤷

Reply:
Perhaps version skew in the sdk versions?